tests/main/help/task.yaml

```
@@ -1,6 +1,6 @@
 summary: Check commands help
 
-systems: [+fedora-*]
+systems: [fedora-*]
```

hmm do we know what the intention here was? does the test work on fedora? perhaps this meant to be 

```suggestion
systems: [-fedora-*]
```

to not run the test on fedora and run everywhere else?

as-is, this will _only_ run the test on fedora which I think is unintentional

---

@anonymouse64 
I'll check that test in a following PR
I'll try to enable in all the systems
Thaks for the review

---

It was meant to be `-fedora-*` because at the time, help would crash on Fedora, due to the difference in the version of the command line helper library.